### ISSUE TITLE
ErrorSummary får ikke fatt i aria-labelledby fra overskriften sin

### ISSUE THREAD

**Issue by krnome** [HelseVestIKT/hviktor#357]
### Beskrivelse

Laget til et hviForm med flere hvi-field og en hvi-error-summary til slutt - får feilmelding i konsoll:

<img width="1136" height="38" alt="Image" src="https://github.com/user-attachments/assets/97126322-4964-4a36-ae31-43f066e20d83" />

```
Designsystemet: Missing accessible name on: <ds-error-summary tabindex=​"-1" class=​"ds-error-summary" role=​"group" aria-labelledby=​":​ds:​1">​…​</ds-error-summary>​ 
Add a heading (h2–h6), or set aria-label or aria-labelledby to provide an accessible name for screen readers.
```

Ser overskriften har fått seg en Id, men får ikke fatt i den?

<img width="817" height="139" alt="Image" src="https://github.com/user-attachments/assets/e40b9e0c-c25c-41b0-b15a-c6a6e1de7c7e" />

Ser det også skjer i konsollen når jeg åpner dokumentasjonen på https://hviktor.no/komponenter/error-summary
